README.md

```diff
@@ -12,23 +13,18 @@ This layer currently provides recipes for the following wolfSSL products:
 - [wolfMQTT lightweight MQTT client library](https://www.wolfssl.com/products/wolfmqtt/)
 - [wolfTPM portable TPM 2.0 library](https://www.wolfssl.com/products/wolftpm/)
 
-This layer currently provides bbappend files for the following open source
-projects:
-
-- [cURL](https://layers.openembedded.org/layerindex/recipe/5765/)
-
-The wolfSSL library recipe is also included in the openembedded
-meta-networking layer, located [here](
-https://github.com/openembedded/meta-openembedded/blob/master/meta-networking/recipes-connectivity/wolfssl/wolfssl_3.14.4.bb).
+The wolfSSL library recipe is also included in the openembedded meta-networking
+layer, located [here](https://github.com/openembedded/meta-openembedded/blob/master/meta-networking/recipes-connectivity/wolfssl/wolfssl_3.14.4.bb).
```

Comment:
As long as we're updating this, can we make this link more generic - pointing to "https://github.com/openembedded/meta-openembedded/blob/master/meta-networking/recipes-connectivity/wolfssl", rather than the specific old 3.14.4?